### DISCUSSION TITLE
Agenda and Minutes for Meeting 23 Feb 2023

### DISCUSSION THREAD

**Discussion by mferrare** [Digital-Law-Lab/Digital-Law-Lab#8] [Meetings]
# Agenda

1. Sirage's pull request
    > @mferrare  merged this PR without reviewing it.  It contains Digital-Law-Lab/docassemble-LLAW33012020S2P06#6 and Digital-Law-Lab/docassemble-LLAW33012020S2P06#15.  @mferrare needs to review and confirm merge is OK
1. Issues to include in next release
    > - @mferrare has created a new milestone **v1.1** in `docassemble-clcsa-01-serivce-directory`.  This is the next release of the app to present to Ippei.
    > - review issues proposed to add to `v1.1`
    > - assign issues
    > - agree on due date
1. Issue review
    > - It's not clear what some of the remaining issues are for.  Review, clarify and update if necessary
    > - discuss assignment of issues (or focus on `v1.1` first)
    > - There are two issues (Digital-Law-Lab/docassemble-clcsa-01-service-directory#7 and Digital-Law-Lab/docassemble-clcsa-01-service-directory#8) that depend on a framework for accessing external data (probably Google Sheets).  Discuss create parent task to set order for tasks and also create additional tasks for this as necessary.
1.  Update repo provisioning document
    > - @mferrare to update Repository Provisioning document according to [this discussion](https://digital-law-lab.slack.com/archives/C03HY6RPPJL/p1677045659285239).
    > - add instructions to rename the main program to `main.yml`
    > - add instructions to add bug report and feature request template to repo
    > - add instructions to change internals of package to reflect new name

**Comment by Sirage-t** [comment]
If time permits, we should also discuss:
- ways to avoid merge conflicts (i.e. should we rebase? always pull latest next? and other questions coz each maintains their own sub branch and we need way to sync all sub branches), 
- "hotfixes", and 
- pull request cycle or schedule (i.e. when should we make PR to next, coz the more changes we make the more conflicts there going to be)

**Reply by mferrare** [reply]
- Before starting to code (for the day?) pull latest `next` into `next_*`
- Raise PRs frequently.  Should be whenever you have a workable `next_*` branch with issues that are closed

**Comment by Sirage-t** [comment]
Re point 4, having instructions is essential, but would you like me to implement an automation for this process in my mighty CLI?

**Reply by mferrare** [reply]
This is a good idea.  I'll make an issue for this and assign it to @Sirage-t
